gilded-rose-llc-frontend/__tests__/index.tests.tsx

```diff
@@ -6,18 +6,23 @@ describe('Home', () => {
   it('should render a sub-heading of admin inventory', () => {
     render(<Home items={[]} />)
 
-    const heading = screen.getByRole('heading', { level: 2, name: "available items" })
+    const heading = screen.getByRole('heading', {
+      level: 2,
+      name: 'available items',
+    })
 
     expect(heading).toBeInTheDocument()
   })
 
   it('should render the name and price of each item on the page', () => {
```

Comment:
Question: Do we have test that tests the page will load the inventory on entry? I believe the admin page also does that.

Reply:
Would you be able to pair tomorrow to explain further, I am a bit confused about the question. Thank you in advance 😄

Reply:
Yes. I can explain and then we can pair tomorrow if needed. Basically, what this test is testing that given a list of items, this page can render all the items, which is fine. However I think we're missing an important behavior which is, there is a network call going out to get those items on entry. If you look at `index.tsx` this is performance the call:
```js
export const getServerSideProps: GetServerSideProps = async () => {
  const gateway = new api.ApiClient(fetch)
  const items = await gateway.getAllItems()
  return { props: { items } }
}
```
If you remove this code, will all the tests still pass?